docs/_components/accordions.md

```diff
@@ -62,76 +68,68 @@ maturity: alpha
   </div>
 </div>
 
-<div class="usa-alert usa-alert-warning" style="margin-bottom: 1rem;">
-  <div class="usa-alert-body">
-    <h3 class="usa-alert-heading">Deprecated</h3>
-    <p class="usa-alert-text">
-      The following accordion has been deprecated since July 2016. Please follow
-      <a href="#deprecated">the instructions</a> to update any existing
-      accordions to the new format.
-    </p>
+<!--
+The following deprecated accordion is kept here for testing purposes for tthe
```

Comment:
extra 't' in 'the'. 

Also this is currently getting rendered on the page because of the !--> at the end.

<img width="688" alt="screen shot 2016-08-15 at 1 36 35 pm" src="https://cloud.githubusercontent.com/assets/776987/17673046/500086be-62ed-11e6-926a-47608134ca49.png">